README.md

```diff
@@ -63,3 +63,12 @@ pricing.**
 [vault-learn-guide]: https://learn.hashicorp.com/tutorials/vault/aws-lambda
 [vault-docs]: https://www.vaultproject.io/docs/platform/aws/lambda-extension
 [vault-aws-iam-auth]: https://www.vaultproject.io/docs/auth/aws
+
+## Testing
+
+If you want to test changes to the lambda extension, you can build and deploy the local version for testing with the Quick Start:
+
+```sh
+make zip
+make quick-start TERRAFORM_ARGS="-var local_extension=true"
+```
```

Comment:
Should we mention the assume_role option somewhere in the readme as well?

Reply:
I have a separate PR in the Vault docs, which seemed like the right place to mention it https://github.com/hashicorp/vault/pull/16677

Reply:
At least for the environment variable. I'll add a note to the README about the terraform variable.

Reply:
Yep, sorry, the terraform variable is what I meant. 👍